custom_components/tahoma/tahoma_device.py

```diff
@@ -169,6 +175,11 @@ async def async_cancel_command(self, exec_id: str):
         """Cancel device command in async context."""
         await self.coordinator.client.cancel_command(exec_id)
 
+    def get_base_device_url(self):
+        """Return base device url."""
+        device_url, _ = self.device.deviceurl.split("#")
```

Comment:
following previous comment:
```suggestion
        device_url, _ = (
            self.device_url.split("#")
            if "#" in self.device_url
            else (self.device_url, "")
        )
```

Reply:
Thanks!

I will hold off a bit more on this PR, since it needs to take into account scenario's where deviceUrl#3 is supported, but where deviceUrl#1 is not supported (climate for example). In this case it should have some kind of a fallback.